### PR TITLE
changed some 'LINK' to ''LinkUSB'

### PR DESCRIPTION
**Pull request by agentmnm**
ATTENTION: It looks like, once a pull request was initialized, it can not be deleted or withdrawn - at least I could not figure out how.

So please close this PR without merging. 

Since changes have to be entered manually into SF master - I did not know that - I might as well go there.

### CODE REVIEW

**Comment by stromnet** [comment]
Hi, thanks for patch!
Not sure about the change though. There is a (although pretty old) device which is just named "Link", which uses a plain serial port afaik: https://www.ibuttonlink.com/products/link45

**Comment by agentmnm** [comment]
Ok, I think I finally understand the manual regarding LINK.

Wherever it says LINK it means a device of the LINK family! And when it 
says LinkUSB it means ONLY specifically the LinkUSB device.

I think I would write in the header explaining that in the manual LINK 
means a device of the LINK family like LinkUSB, LinkHubE, Link45 etc.


So how do I retract the pull request? Or is it easier if you reject that 
request? And then let me try again


P.S. I am not sure if 'reply' is enough, so you recieve the mail, so I 
did now try to 'reply all' - not sure, again I am not experienced with 
github, sorry

**Comment by stromnet** [comment]
1 email received, and PR seems updated once, so seems to work :)

You can either just create an additional commit, or perhaps close the PR yourself (should be doable).
If it where me, I'd probably alter the code, commit with --amend and push with --force. 

Note: github is not the official home of owfs (yet?), so any PR will have to be manually merged to master outside of github anyway.

**Comment by agentmnm** [comment]
Ok I did not know that changes have to be manually merged at SF, so I might as well go there. 
I updated the message for the PR to go ahead and close without merging, since it looks like, that I cannot close a PR myself. In my account the PR is gone and I cannot edit OWFS master obviously ;)

Thanks

**Comment by agentmnm** [comment]
thanks.
changes and additions to the manual will come from me - hopefully soon - 
to sourceforge
